image-index.md

```diff
@@ -35,7 +35,6 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
     Implementations MUST support at least the following media types:
 
     - [`application/vnd.oci.image.manifest.v1+json`](manifest.md)
-    - [`application/vnd.oci.artifact.manifest.v1+json`](artifact.md)
```

Comment:
Maybe just move this line to `SHOULD support`, then no need to remove the entire artifact spec?

Reply:
I think this would be a good change regardless. Basically, all registries _MUST_ support image manifests, and SHOULD (or even MAY) support the artifact manifest.

Without this change, a registry could choose to _only_ support the artifact manifest, which would probably only cause a deeper fracture in the ecosystem.

Reply:
Without any change, registries MUST support _both_ the artifact and image manifests.

Reply:
I think this would lead to a great deal of confusion for OCI 1.1 compliance and support for artifacts. With so many registries already adopting Artifact in it's RC form, what is gained by weakening this MUST to SHOULD?

Reply:
@AaronFriel it lets registries adopt the references API (much more valuable imo) and claim 1.1 conformance. If they have to adopt both to be 1.1, they might prefer not to invest in either.

Reply:
> Without any change, registries MUST support _both_ the artifact and image manifests.

That's not how I read "MUST support either of these media types". Am I missing something?

Reply:
> @AaronFriel it lets registries adopt the references API (much more valuable imo) and claim 1.1 conformance. If they have to adopt both to be 1.1, they might prefer not to invest in either.

What utility do references have absent artifacts? If registries can claim conformance with OCI 1.1 but no one can upload an sbom, NPM package, cat gif, etc., I don't see the point of OCI 1.1.

Reply:
Clients have been successfully uploading SBOMs, NPM packages, cat pictures, etc., for a couple years now at least under 1.0.

1.1 will codify some of those patterns and augment them with a first-class supported API that doesn't require polluting tag hacks.